src/components/deck/deck.js

```diff
@@ -134,7 +136,17 @@ const Deck = forwardRef(
         ? {}
         : {
             left: () => stepBackward(),
-            right: () => stepForward()
+            right: () => stepForward(),
+            ...(overviewMode && {
+              [KEYBOARD_SHORTCUTS.TAB_FORWARD_OVERVIEW_MODE]: () =>
```

Comment:
We should attach these to the `DefaultDeck` component or split that out into an `OverviewMode` component that wraps `DefaultDeck`.